docs/cadence_migration_guide/index.md

```diff
@@ -6,19 +6,60 @@ sidebar_label: Cadence 1.0 Migration Guide
-- [Learn how to update your Non-Fungible Token contracts for Cadence 1.0](./nft-guide.mdx)
-- [Learn how to update your Fungible Token contracts for Cadence 1.0](./ft-guide.mdx)
-- [Check out the Cadence language 1.0 docs](https://cadencelang.dev/docs/1.0/)
+To ensure your contracts are fully operational with Cadence 1.0, follow these essential steps:
+
+1. **Understand the Changes:** Start by familiarizing yourself with the [changes](https://forum.flow.com/t/update-on-cadence-1-0/5197) to identify how they might affect your code.
+2. **Modify Your Code:** Update your contracts, transactions, and scripts in accordance with the new changes.
+3. **Test Your Code:** Use the [Cadence 1.0 Emulator](https://www.loom.com/share/4467610b7beb4ebbaabed6b430dc25c4?sid=14ecb3e7-e933-409c-a6a4-add40c6971d0) to test your code by running:
+
+Linux/macOS
+
+```bash
+
+sudo sh -ci "$(curl -fsSL https://raw.githubusercontent.com/onflow/flow-cli/master/install.sh)" -- v1.12.0-cadence-v1.0.0-M4-2
+
+```
+
+Windows (in PowerShell):
+
+```
+
+iex "& { $(irm 'https://raw.githubusercontent.com/onflow/flow-cli/master/install.ps1') } v1.12.0-cadence-v1.0.0-M4-2
+```
+
+1. **Stage**: A new **_Staging process_** will be released in the coming weeks that checks if your updated code is compatible with Cadence 1.0. Complete this [form](https://docs.google.com/forms/d/e/1FAIpQLSfprZJLPSEAS6H7_oL0j6bzetDzkHPmDZHYAGgqAAOAdLDKqw/viewform) to stay informed about updates and receive recommendations tailored to your code.
 
-## Get Help With Migrating Your Dapp to Cadence 1.0
+### Resources
 
-If you have any questions related to Cadence 1.0 migration, or if you need help, please contact the Flow development community on [developer-questions Discord channel](https://discord.com/channels/613813861610684416/621847426201944074). Please use “C1.0 upgrade” at the beginning of the message to help us categorize questions related to Cadence 1.0.
+**Guides & Docs**
 
-The Flow team has started hosting regular "Cadence 1.0 upgrade office hours" events anyone can join to ask questions and get help with their dapp upgrade. These sessions are scheduled in [Flow Webinars & Events calendar](https://calendar.google.com/calendar/u/0?cid=Y180Nzk3OGY1Y2Q5ZGE2MzZjYWRjNmI4NDczMTAyYjUwOTJjMWE4NjVkZDAxMDU1ODM5M2VjYjdmOWZkMGM5YWQwQGdyb3VwLmNhbGVuZGFyLmdvb2dsZS5jb20) and announced on [flow-events Discord channel](https://discord.com/channels/613813861610684416/1050190147100102787).
+For assistance with the most common updates developers will face, explore these resources:
+
+- Upgrading [Capabilities](https://academy.ecdao.org/en/catalog/tutorials/capability-controllers)
+- Upgrading [Entitlements](https://academy.ecdao.org/en/catalog/tutorials/entitlements)
+- Migration Guide for [Fungible Tokens](https://cadence-lang.org/docs/cadence_migration_guide/ft-guide)
+- Migration Guide for [NFTs](https://cadence-lang.org/docs/cadence_migration_guide/nft-guide)
+
```

Comment:
```suggestion
Migration Guide for the [Core Protocol Smart Contracts](./core-contracts-guide.mdx)
```
You should update all the links in here that link to other pages in cadence-lang to use relative links like this instead of full links.

Reply:
Done @joshuahannan!